contents/docs/session-replay/mobile.mdx

```diff
@@ -11,134 +11,107 @@ Mobile Session Recording allows you to record user sessions on mobile devices. T
+    .build()
 ```
 
-
 # Recording specific configuration
 
-```swift
-// The values shown are the defaults
-let config = PostHogRecorderConfig(
-    // Whether to enable recording. If false, all other options are ignored
-    screenRecordingEnabled: true,
-    // Whether to enable log recording. Logs with OSLog will be automatically recorded or you can log directly with PostHog.shared.recorder.log()
-    logRecordingEnabled: true,
-    // Records network requests via swizzling of NsUrlSession.
-    networkRecordingEnabled: true,
-    // What level of redcaction to perform
-    // -> .selective = only redact manually specified views with an accessibilityId matching what is specified in `redactionTags`.
-    // -> .automatic = redact all views with the name matching `redactionViews` and all classes matching `redactionClasses`
-    // -> .wireframe = renders a "wireframe" of the view, without any text or images
-    redactionMode: .selective,
-    redactionTags: ["ph-no-capture"],
-    redactionViews: PostHogRecorderConfig.DefaultRedactionViews,
-    redactionClasses: PostHogRecorderConfig.DefaultRedactionClasses,
-    // Hosts to ignore when recording network requests. By default the PostHog API host is ignored
-    ignoreNetworkHosts: []
-)
+The SDK configurations.
+
+```android_kotlin
+val config = PostHogAndroidConfig(apiKey = POSTHOG_API_KEY).apply {
+    // enable session recording, requires enabling on the project configuration as well.
+    sessionReplay = true
+    // all texts are masked (default is enabled)
+    sessionReplayConfig.maskAllTextInputs = false
+    // all images are placeholders (default is enabled)
+    sessionReplayConfig.maskAllImages = false
+    // capture logs automatically (default is enabled)
+    sessionReplayConfig.captureLogcat = true
+}
 ```
 
+The View declaration tag, the image will be a placeholder if the tag is set.
+
+```xml
+<ImageView
+    android:id="@+id/imvProfilePhoto"
+    android:layout_width="200dp"
+    android:layout_height="200dp"
+    android:tag="ph-no-capture"
+/>
+```
 
+# Limitations
+
+* Requires the PostHog Android SDK version >= [3.1.0](https://github.com/PostHog/posthog-android/releases/tag/3.1.0) - use the latest version.
+* Requires Android API >= 26, otherwise it's a NoOp.
+* Not compatible with Jetpack Compose.
+* Generates wireframe representations of a users screen, not a video recording or a screenshot.
+  * Custom views are not fully supported.
+* WebView is not supported, a placeholder will be shown.
+* React Native and Flutter for Android aren't supported.
+* [Authorized Domains for Replay](https://us.posthog.com/settings/project-replay#replay-authorized-domains) has to be disabled, for now, or create a new testing project to not disrupt your current project configuration.
+* Let us know when you're ready to view the recordings. While replay is in beta there's still some setup on our side before you can playback the sessions.
```

Comment:
I think this should be a bigger point, maybe a call out at the end of the setup? 

Also, would be nice to add a link how you want them to contact you, like the support panel: https://us.posthog.com#panel=support%3Afeedback%3Asession_replay%3Alow